### DISCUSSION TITLE
battery fails to launch on mac M1

### DISCUSSION THREAD

**Discussion by GHT52** [community/community#82929] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

![Capture d’écran 2023-12-28 à 18 27 35](https://github.com/community/community/assets/11530415/a3411af1-d4cc-480d-8cd5-8c8c1a6d65a1)
Is there a workaround?
Regards,
Gérald

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by DNXEMPIRE-1** [comment]
Shut the laptop for Atleast 1hour

**Comment by GHT52** [comment]
I did shut it down for a whole night. Still same error message.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GHT52 ! 

We're happy you're here. The Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Mac, like the [Mac Support Communities](https://discussions.apple.com/community/mac). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
